mmv1/products/bigquery/Job.yaml

```diff
@@ -392,6 +401,11 @@ properties:
                 enum_values:
                   - 'LAST'
                   - 'FIRST_SELECT'
+          - name: 'continuous'
```

Comment:
Could you add modification of this field to an update test as well? (Ideally including setting it from true to false (or to unset))

Reply:
This resource doesn't support Update: https://github.com/hashicorp/terraform-provider-google/blob/63cdd6a13df80017b27cde4cd7488cf365cc19ee/google/services/bigquery/resource_bigquery_job.go#L1220

Public API: https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs

Reply:
ah, yep! Thanks!